src/scaladoc/scala/tools/nsc/doc/model/MemberLookup.scala

```diff
@@ -128,11 +152,15 @@ trait MemberLookup extends base.MemberLookupBase {
   }
 
   lazy val javaVersion: Int =
-    System.getProperty("java.specification.version").split('.').take(2).map(_.toIntOption) match {
-      case Array(Some(1), Some(n)) => n   // example: 1.8.0_242
-      case Array(Some(n))          => n   // example: 14
-      case Array(Some(n), _)       => n   // example: 11.0.7
-      case _                       => 8   // shrug!
+    global.settings.releaseValue
+      .getOrElse(System.getProperty("java.specification.version"))
+      .split('.')
+      .take(2)
+      .map(_.toIntOption) match {
+      case Array(Some(1), Some(n)) => n // example: 1.8.0_242
+      case Array(Some(n))          => n // example: 14
+      case Array(Some(n), _)       => n // example: 11.0.7
+      case _                       => 8 // shrug!
```

Comment:
Should use `scala.util.Properties` instead
```
scala> util.Properties.javaSpecVersion
val res0: String = 23

scala> util.Properties.isJavaAtLeast(11)
val res1: Boolean = true
```

Reply:
I've pushed changes that replace `System.getProperty("java.specification.version")` with `scala.util.Properties.javaSpecVersion`. I couldn't replace version checks in this file e.g ...

```
if (javaVersion < 11) {
      s"https://docs.oracle.com/javase/$javaVersion/docs/api"
```

with `isJavaAtLeast` calls. `javaSpecVersion` seems to be pinned to the version of the JVM I'm running the scaladoc tool with, whereas a lot of these checks are for what version of JVM we're targeting with the scaladoc tool (specified with `-release`).